runtime/account_test.go

```diff
@@ -1289,24 +1289,16 @@ func TestRuntimePublicKey(t *testing.T) {
 			storage: storage,
 		}
 
-		value, err := executeScript(script, runtimeInterface)
-		require.NoError(t, err)
```

Comment:
Or maybe add another test for the above behavior

Reply:
I can't actually think of a way to use references this way, since arrays are copied by value rather than by reference. Can you think of a way to cause the behavior you are describing?

Reply:
We can try to get a reference to the publickey field `var publickeyRef = &pubKey.publickey as &[UInt8]` and try to update it via reference.
i.e: Below is a version that uses a user-defined struct. I think we can try to do the same with built-in public key.
```kotlin
pub fun main() {
  var pubKey = PublicKey()

  var publickeyRef = &pubKey.publickey as &[UInt8]
  publickeyRef[0] = 3

  log(pubKey.publickey)
}

pub struct PublicKey {
  pub var publickey: [UInt8]

  init() {
    self.publickey = [1, 2]
  }
}
```

Reply:
Aha yea I see, I was thinking you would have to pass the byte array reference into the constructor, but this makes more sense.